tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.html

```diff
@@ -0,0 +1,30 @@
+<div id="editor">
+	<p>Foo</p>
+</div>
+
+<script>
+
+	if ( bender.tools.env.mobile ) {
+		bender.ignore();
+	}
+
+	CKEDITOR.replace( 'editor', {
+		height: 300
+	} );
+</script>
+
+<button onclick="start()" >Start.</button>
+<script>
+	function start() {
```

Comment:
Why so complicated? 😱 

```javascript
function start() {
    var ed = CKEDITOR.instances.editor;
    ed.focus();
    ed.getSelection().removeAllRanges();
}
```
seems to do the trick on Chrome and Firefox, so I suppose that it will also work in other browsers.